### DISCUSSION TITLE
parentContainer[ROLL_UP_REGISTRATIONS] is not a function

### DISCUSSION THREAD

**Discussion by project-ASTA** [medusajs/medusa#10381] [Q&A]
The error is occurring after i update to new versions, but I'm not sure, but it works perfectly fine the same packages and everything in my dev environment but when i host on railway i get this error, when i try to run custom workflows

**Comment by srindom** [comment]
Hi, can you provide a minimal reproduction of this so we can investigate.

**Comment by umarluqman** [comment]
> The error is occurring after i update to new versions, but I'm not sure, but it works perfectly fine the same packages and everything in my dev environment but when i host on railway i get this error, when i try to run custom workflows

are you able to fix it? I'm encountering the exact same issue, hosting in railway and also trying to run custom workflows

**Comment by praveen-ak** [comment]
Hi, I am also facing the same issue, 
Issue comes on workflow code setup

local works fine, issue only on production

currently medusa 2 using awilix package for creating container,  version is 8.0.1

if update the awilix package, 
this issue will be sorted. ROLL_UP_REGISTRATIONS

But I am facing another issue on 
message: "Could not resolve 'query'.\n\nResolution path: query",
      name: 'AwilixResolutionError',
      stack: "AwilixResolutionError: Could not resolve 'query'.\n" +

same on workflow

SO THE ISSUE IS AWILIX in WORKFLOW

if one issue fix by upgrading awilix version, that will throw another issue

So Please give proper solution for awilix package 

Thanks

**Comment by aaa3334** [comment]
Have the same issue - hosting on railway - works on local but get parentContainer[ROLL_UP_REGISTRATIONS] is not a function when I try and call a custom workflow/module on railway (here it is an update function to the database)

**Comment by aaa3334** [comment]
Found the solution digging through reddit - change the absolute src/* paths to relative .../... -> in production the absolute paths do not work but they do in dev

**Reply by Op-Pautu** [reply]
@aaa3334 can u link where u got that if u still remember?
